src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/CodeMetricsComputationTests.cs

```diff
@@ -503,7 +503,7 @@ class C4 { public static C2 MyC2 = null; }
 ";
 
             var expectedMetricsText = @"
-Assembly: (Lines: 12, ExecutableLines: 3, MntIndex: 95, CycCxty: 5, CoupledTypes: {C1, C2, C3, C4}, DepthInherit: 2)
+Assembly: (Lines: 12, ExecutableLines: 3, MntIndex: 96, CycCxty: 5, CoupledTypes: {C1, C2, C3, C4}, DepthInherit: 2)
```

Comment:
I have replaced with the value given but I have no idea if the value makes more sense or not.

Given these changes, it may be better to wait for a major bump before merging except if we consider this was an important bug.